src/applications/vaos/tests/e2e/vaos-helpers.js

```diff
@@ -150,6 +150,28 @@ function appointmentSubmittedTest(client) {
   return client;
 }
 
+function showMoreTest(client) {
+  client
+    .click('.additional-info-button.va-button-link')
+    .waitForElementPresent('.additional-info-content', Timeouts.slow);
```

Comment:
I think we would like to be able to test that some text shows up in the content block after show more is clicked. Not sure how easy that is, but otherwise I don't think we'd know if fetching that message is broken or not.

Reply:
I think the additional info section doesn't display at all when the fetch fails. I can verify this and add the check for some data that displays in that section. My preference is to not have the test dependent on the data if possible.

Reply:
Yeah, if that div isn't showing until the request finishes, that's fine. I just want to make sure we can tell the difference between the fetch messages call failing and it succeeded.